docs/tutorials/Hosting_on_Heroku.md

```diff
@@ -0,0 +1,197 @@
+
+### Prepping required components
+
+> INFO
+> If you don't have git configured, you can read [the Git reference](https://git-scm.com/book/en/v2/Getting-Started-First-Time-Git-Setup)
+
+To configure your app locally, first clone or fork the repository and install its dependencies.
+
+```bash
+# Clone the respository
+git clone https://github.com/discord/heroku-sample-app.git
+
+# Enter into the directory
+cd heroku-sample-app
+
+# Install the dependencies
+npm install
+```
+
+After your dependencies are installed, you'll create your app and configure Heroku.
+
+### Creating your Discord app
+
+In the `heroku-baker` directory, there is a `.sample.env` file that contains the credentials baker bot needs to run:
+
+```
+TOKEN=
+GUILD_ID=
+CLIENT_ID=
+```
+
+These credentials will be added to your Heroku app later in the tutorial, or to a new `.env` file if you're developing locally.
+
+> INFO
+> Enabling Developer Mode in your Discord client will let you right-click fetch the IDs for your guild (`GUILD_ID`) and bot application (`CLIENT_ID`).
+
+To create your Discord app, go to the [app configuration](https://discord.com/developers/applications) and click **New Application** in the upper-right corner.
+
+Then click on the **Bot** tab and create a bot. Generate the bot token and save it in a safe place.
+
+![Discord bot token](heroku-token.png)
+
+After your bot is created, go to **OAuth2** on the sidebar and copy the **Client ID** for later.
+
+Then click **URL Generator**. Add the `applications.commands`, `Manage Roles` and `bot` scopes.
+
+![OAuth2 URL Generator](cloudflare-url-generator.png)
+
+To install your app, copy and paste the URL in your browser and follow the installation flow. Make sure to install the app into a server where you can test and develop.
+
+Once your app is installed copy the **Guild ID** for the server where you installed your app.
+
+## Setting up Heroku
+
+The first thing you should do is set up an account for Heroku.
+
+Head over to https://id.heroku.com/login and log in to your account, or [set up a new account](https://signup.heroku.com/) if you don’t have one.
+
+![Heroku login](heroku-login.png)
+
+After logging in, at the top right corner of your account dashboard, click New > Create new app. You can then name it whatever you want.
+
+### Adding a Procfile
+
+For your app to work with Heroku, you'll need to add a `Procfile`. **Create a new file called `Procfile`** where you will add a short script that Heroku will run when starting up your app.
+
+> INFO
+> For folks following the tutorial and using the [sample repo](https://github.com/discord/heroku-sample-app), the `Procfile` will already be configured appropriately. Otherwise, the script in your `Procfile` will vary depending on what language you're coding in.
+
+```bash
+# Procfile contents for bakerbot (startup)
+worker node index.js
```

Comment:
```suggestion
worker: node index.js
```
Missing the `:`, right? [Refs](https://devcenter.heroku.com/articles/procfile#procfile-format)